### PR TITLE
Alex

### PR DESCRIPTION
**Pull request by Aleksei12942**

### CODE REVIEW

File: app/controllers/concerns/training_plan_concern.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module TrainingPlanConcern
+  extend ActiveSupport::Concern
+
+  def name(training)
+    name = Client.find_by(id: training.client_id).first_name + ' ' + Client.find_by(id: training.client_id).second_name
+    name
+  end
+
+  def sets(training, current_user)
+    sets = Kit.where(training_id: training.id, user_id: current_user.id).map do |kit|
```
Comment (brutu33) [inline app/controllers/concerns/training_plan_concern.rb]: it's hard to understand what going on here

File: app/controllers/exercises_controller.rb [hunk 2]
```diff
@@ -47,7 +42,11 @@ def update
       @name = (ExerciseType.find_by(id: @exercise.exercise_type_id)).name
       render plain: "#{@name}"+ " | " + "повторений: " + "#{@exercise.repeats}"+ "," + "подходов: " "#{@exercise.approach}"
     else
-      render plain: "ahuet project"
+      @count = @@count
```
Comment (brutu33) [inline app/controllers/exercises_controller.rb]: @@count - what is it? Why you use it?

File: app/controllers/visitors_controller.rb [hunk 3]
```diff
@@ -8,4 +8,4 @@ def index
       redirect_to calendar_index_path
     end
   end
-end
+end
```
Comment (brutu33) [inline app/controllers/visitors_controller.rb]: end file newline

File: app/controllers/concerns/training_plan_concern.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+module TrainingPlanConcern
+  extend ActiveSupport::Concern
+
+  def name(training)
+    name = Client.find_by(id: training.client_id).first_name + ' ' + Client.find_by(id: training.client_id).second_name
```
[the remark this answers is not in the document]
Comment (Aleksei12942) [reply app/controllers/concerns/training_plan_concern.rb]: Оf cause I want to rewrite this long string every time, when I want get client's full name)

File: app/controllers/concerns/training_plan_concern.rb (hunk 1 above)
Comment (Aleksei12942) [reply app/controllers/concerns/training_plan_concern.rb]: Getting all kits, that belongs to current training and current user

File: app/controllers/exercises_controller.rb (hunk 2 above)
Comment (Aleksei12942) [reply app/controllers/exercises_controller.rb]: This variable is used to make divs with unique IDs

File: app/controllers/calendar_controller.rb [hunk 5]
```diff
@@ -13,12 +13,16 @@ def download
...
           start: date[0...10].to_s + ' ' + date[10...15],
-          price: training.price,
-          description: training.description,
-          status: training.status
+          id: training.id
       }
       @data << info
     end
+    google = {
+        googleCalendarId: 'k9qatn9fshgu56rhtccrec0dao@group.calendar.google.com',
```
Comment (Equinox-01) [inline app/controllers/calendar_controller.rb]: It's not good

File: app/controllers/trainings_controller.rb [hunk 6]
```diff
@@ -16,24 +17,17 @@ def index
...
   end
 
   def new
     @training = current_user.trainings.build
     @list = client_list(current_user)
+    date = params[:date][0...10]
+    @day = date[8].to_i == 0 ? date[9] : date[8..9]
+    @month = date[5].to_i == 0 ? date[6] : date[5..6]
+    @year = date[0..3]
   end
```
Comment (Equinox-01) [inline app/controllers/trainings_controller.rb]: Magic numbers ? :-)

**Pull request merged**